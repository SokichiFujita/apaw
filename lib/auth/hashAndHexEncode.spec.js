const hashAndHexEncode = require('./hashAndHexEncode');

describe('hashAndHexEncode', () => {
  it('should create a hexidecimal string of input object values', () => {
    const hashedString = hashAndHexEncode({
      firstName: 'john',
      lastName: 'foderaro',
    });
    expect(hashedString).toMatchSnapshot();
  });
});
