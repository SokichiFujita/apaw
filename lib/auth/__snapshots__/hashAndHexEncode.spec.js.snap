// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`hashAndHexEncode should create a hexidecimal string of input object values 1`] = `"629aa885c8f8256c5d931fe161a5a2473248709ae7239d37628a98d8c63a5106"`;
